test/membrane_rtc_engine/engine_test.exs

```diff
@@ -330,6 +330,7 @@ defmodule Membrane.RTC.EngineTest do
   end
 
   describe "Engine linking test" do
+    @tag :skip
```

Comment:
Why do we skip this test?

Reply:
Raw format is no longer an option in engine but I am gonna introduce required changes in a separate PR